### ISSUE TITLE
Attempted to import polyline (stream network) `Import existing Scratch Vector Feature Class` and fails to import.

### ISSUE THREAD

**Issue by joewheaton** [Riverscapes/QRiS#141]
I tried importing the Stream Network line using the same flawed process described in #140, but unlike that where it at least successfully imported, this failed to import, but created an empty class in the Layers TOC. The OGR warning is thrown:
```
2022-10-22T11:34:07     WARNING    Cannot reopen C:/0_RS/QRiS/Tester0-1-5a\Temple__Spawn\scratch\feature_classes.gpkg: Cannot find layer NHDFlowline.
```  

https://youtu.be/9FF8SgMRwEM

**Comment by philipbaileynar** [comment]
I have experienced this too.

This is an example of where open source sucks.... there's no "copy feature class" routine available that we can find. You have to write your own.... There's a command line `ogr2ogr` tool, but it's not available from inside Q.

**Comment by KellyMWhitehead** [comment]
This has been fixed for version 0.1.16.
